scripts/deploy.sh

```
@@ -1,96 +1,45 @@
-if [ -z $IDLE_PID ]
-then
-  echo "> 현재 구동중인 애플리케이션이 없으므로 종료하지 않습니다."
-else
-  echo "> kill -15 $IDLE_PID"
-  kill -15 $IDLE_PID
-  sleep 20
-fi
+echo "> 애플리케이션 시작"
+nohup java -jar \
+  -Duser.timezone=Asia/Seoul \
+  $JAR_PATH \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)